### PR TITLE
[WIP] Minimalist components to introduce kinto.js

### PR DESCRIPTION
**Pull request by leplatrem**
- [x] Publish kinto.js on npm
- [x] Add kinto.js to deps
- [x] List + form components
- [x] Test record creation
- [ ] Sync button
- [x] Plug kinto.js

### CODE REVIEW

File: test/index_test.js [hunk 1]
```diff
@@ -1,10 +1,47 @@
...
+
+  it("contains an input field and a submit button", () => {
+    const rendered = TestUtils.renderIntoDocument(<Form/>);
+    const selector = "button[type='submit'],input[type='text']";
+    const nodes = React.findDOMNode(rendered).querySelectorAll(selector);
+    expect(nodes.length).to.equal(2);
+  });
+
+  it("triggers a creation on submit", () => {
+    const rendered = TestUtils.renderIntoDocument(<Form/>);
```
Comment (n1k0) [inline test/index_test.js]: Seems you could factorize that line in a `beforeEach`.

File: scripts/components/App.js [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+    );
+  }
+}
+
+
+export class Form extends React.Component {
+  onFormSubmit(event) {
+    event.preventDefault();
+    var record = {item: event.target.item.value};
+    console.log(record);
```
Comment (n1k0) [inline scripts/components/App.js]: Don't forget to remove this line :)

File: scripts/components/App.js [hunk 3]
```diff
@@ -1,5 +1,42 @@
...
+      }</ul>
+    );
+  }
+}
+
+
+export class Form extends React.Component {
+  onFormSubmit(event) {
+    event.preventDefault();
+    var record = {item: event.target.item.value};
```
Comment (n1k0) [inline scripts/components/App.js]: Nit: how about `name` or `label` instead of `item`?

File: test/index_test.js [hunk 4]
```diff
@@ -1,10 +1,47 @@
...
+    const node = React.findDOMNode(rendered);
+    TestUtils.Simulate.submit(node);
+    // XXX :)
```
Comment (n1k0) [inline test/index_test.js]: It's probably time to update some component state ;)

**Comment by leplatrem** [comment]
With sinon... 

```
TypeError: modules[moduleId] is undefined
  at /home/mathieu/Code/Mozilla/kinto-react-boilerplate/tests.webpack.js:20:0 <- webpack:/webpack/bootstrap 2402b766d5138bfb3db2:19:0
Firefox 39.0.0 (Ubuntu 0.0.0): Executed 0 of 0 ERROR (0.253 secs / 0 secs)

```

**Comment by n1k0** [comment]
> TypeError: modules[moduleId] is undefined

You need to readd it to `package.json` :)

**Pull request merged**